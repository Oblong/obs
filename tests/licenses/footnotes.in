Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
Upstream-Name: footnotes

Files: *
Copyright: Alfred E. Neuman
License: MIT or Blarg

License: Blarg
 Blarg denies you the right to use anything, ever.
 So there.
