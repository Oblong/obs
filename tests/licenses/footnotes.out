Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
Upstream-Name: footnotes

Files: *
Copyright: Alfred E. Neuman
License: MIT or Blarg
